static/js/components/NotificationContainer_test.js

```diff
@@ -91,4 +91,50 @@ describe("NotificationContainer component", () => {
       hiddenNotifications: new Set(["message1"])
     })
   })
+
+  it("shows cms notifiction", async () => {
+    const { inner } = await render({
+      ui: {
+        userNotifications: {
+          "cms-site-wide-notification": {
+            type:  ALERT_TYPE_TEXT,
+            props: { text: "Cms Notifictaion", persistedId: 1 }
+          }
+        }
+      }
+    })
+
+    const alerts = inner.find("Alert")
+    assert.lengthOf(alerts, 1)
+    const cmsNotifictaion = alerts.at(0)
+    assert.equal(cmsNotifictaion.prop("children").type, TextNotification)
```

Comment:
We this `children` prop multiple times below. Let's store it in a variable called `notificationContents` or something like that